src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialNewFilesPage.java

```diff
@@ -43,71 +40,60 @@
   @FindBy(css = "input[name='wpDestFile']")
   private WebElement fileNameInput;
 
-  public SpecialNewFilesPageObject(WebDriver driver) {
-    super();
-  }
-
-  public void addPhoto() {
+  public SpecialNewFilesPage addPhoto() {
     scrollAndClick(addPhotoButton);
-    PageObjectLogging.log(
-        "ClickAddPhotoButton",
-        "Add photo button clicked",
-        true
-    );
+    PageObjectLogging.log("ClickAddPhotoButton", "Add photo button clicked", true);
+
+    return this;
   }
 
-  public void clickUploadButton() {
+  public SpecialNewFilesPage clickUploadButton() {
     scrollAndClick(uploadFileInput);
-    PageObjectLogging.log(
-        "ClickOnUploadaPhoto",
-        "Click on upload a photo button",
-        true
-    );
+    PageObjectLogging.log("ClickOnUploadaPhoto", "Click on upload a photo button", true);
+
+    return this;
   }
 
-  public void setFileName(String fileName) {
+  public SpecialNewFilesPage setFileName(String fileName) {
     fileNameInput.clear();
     fileNameInput.sendKeys(fileName);
+
+    return this;
   }
 
-  public void clickOnMoreOptions() {
+  public SpecialNewFilesPage clickOnMoreOptions() {
     moreOrFewerOptions.click();
     waitForValueToBePresentInElementsCssByCss("div.options", "display", "block");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "display" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)